docs/public.rst

```diff
@@ -99,6 +102,52 @@ Finally, the message is decrypted (regardless of how the nonce was generated):
     plaintext = alice_box.decrypt(encrypted)
 
 
+nacl.public.SealedBox
+~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~nacl.public.SealedBox` class encrypts messages addressed
+to a specified key-pair by using ephemeral sender's keypairs, which
+will be discarded just after encrypting a single plaintext message.
+
+This kind of construction allows anonymously sending messages, which
```

Comment:
Would "without providing any kind of cryptographic proof of sender's authorship" be better than "anonymously"?